src/NServiceBus.Core.Tests/ApprovalFiles/APIApprovals.ApproveNServiceBus.approved.txt

```diff
@@ -1740,10 +1746,13 @@ namespace NServiceBus.Pipeline
     }
     public class MessageHandler
     {
+        public MessageHandler() { }
+        [System.Obsolete("Use \'MessageHandler<THandler, TMessage>\' instead. Will be removed in version 11.0" +
```

Comment:
Should this be using the new [Particular.Obseletes](https://particularsoftware.slack.com/archives/C0BHQ5PJ8/p1759523456470359)?